src/components/fields/date-field/types.ts

```diff
@@ -11,6 +12,7 @@ export interface IDateFieldValidationRule extends IYupValidationRule {
 	minDate?: string | undefined;
 	maxDate?: string | undefined;
 	excludedDates?: string[] | undefined;
+	withinDays?: TComponentOmitProps<IWithinDaysRule, "dateFormat"> | undefined;
```

Comment:
no need to use `TComponentOmitProps` because that is for omitting FEE schema keys

```suggestion
	withinDays?: Omit<IWithinDaysRule, "dateFormat"> | undefined;
```